pages/support.tsx

```diff
@@ -0,0 +1,301 @@
+import React from 'react';
+import styled from 'styled-components';
+import Head from 'next/head';
+import Footer from '@/common/components/home/Footer';
+import Image from 'next/image';
+
+const Heading = styled.h2`
```

Comment:
Not required for this PR: We're using tailwind for styling everywhere, rather than styled-components or inline-styling (when possible).

You can still make reusable components using tailwind, e.g.

```
function Paragraph(props: {children: React.ReactNode, className?: string}): JSX.Element {
   return (
      <p className={classNames("text-base", props.className)}>{props.children}</p>
   )
}
```

Reply:
Everything has been switched over to this style